### ISSUE TITLE
openHABian integration

### ISSUE THREAD

**Issue by ThomDietrich** [jens-maus/RaspberryMatic#217]
Hello @jens-maus,

I am the main developer of [openHABian](http://docs.openhab.org/installation/openhabian.html), the openHAB system for Raspberry Pi and other Ubuntu/Linux based systems. We offer many [optional components](http://docs.openhab.org/installation/openhabian.html#optional-components) to be installed on the system in parallel to connect openHAB to hardware components like Homematic components.

We would be interested to include RaspberryMatic as an alternative/replacement for homegear. Sadly you only provide an image. How would I be able to deploy a RaspberryMatic on an existing Debian system? Do you have any prior experience with that?

Best regards and in hope for a great cooperation, 
Thomas

**Comment by jens-maus** [comment]
RaspberryMatic is meant to be an own distribution which is based on the buildroot Linux operating system which is targeted for lightweight embedded devices instead of using a full bloated Debian/Raspbian system. As such, our target is to provide a self-contained image which can directly be used to replace a CCU device sold by the makers of HomeMatic (eQ3). On top of that so-called Addons can then be used to add additional components or connect to other systems like openHAB (which is currently not directly supported).

RaspberryMatic itself is using components from the OCCU framework (the open source ecosystem of the CCU/HomeMatic device sold by eQ3, see https://github.com/eq-3/occu).

If you are, however, searching for a way to install the CCU functionality using Debian packages there is a separate project (piVCCU - https://github.com/alexreinert/piVCCU/) which uses LXC container to emulate a CCU and directly run the official firmware on a RaspberryPi in a LXC container environment. While this seems to be a very nice way to emulate/simulate a CCU2 device directly using OCCU components can also be a viable way. In fact, it might even be possible to extract all the necessary components from RaspberryMatic and create Debian packages from it... But this is another story ;)
